pkg/remote/aws/db_instance_supplier.go

```diff
@@ -43,8 +48,7 @@ func (s DBInstanceSupplier) Resources() ([]resource.Resource, error) {
 	resourceList, err := listAwsDBInstances(s.client)
 
 	if err != nil {
-		logrus.Error(err)
-		return nil, err
+		return nil, NewBaseListError(err, resourceaws.AwsDbInstanceResourceType, resourceaws.AwsDbInstanceResourceType)
```

Comment:
Hmm it's sad to have to send two types, 90% of the time this is useless as it only concern resources that require multiples levels of listing. Any idea how to handle this in a better way ?

Reply:
Maybe using a struct to pass context

```go
NewBaseListError(err, ErrorListContext{supplierType: resourceaws.AwsDbInstanceResourceType})
NewBaseListError(err, ErrorListContext{supplierType: resourceaws.AwsDbInstanceResourceType, erroredListType: resourceaws.AwsDbInstanceResourceType })

```

Reply:
I could also make a special constructor if you prefer.